docs/TRANSPORTATION.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Transportation
+description: Overview of the Transportation Program
+permalink: /transportation
+---
+
+In order to ensure all Detroit residents have access to the COVID-19 testing site, a transportation service was stood up to pick-up residents at their home, take them through the testing site, drop residents back off at their residence, and then sanitize the vehicle.
+
+How it works:
+
+1. Residents that have met the qualification requirements contact the COVID-19 Test Site call center to request an appointment. If a resident requests transportation during this call, the call center representative connects the resident to the transportation call center for the resident to book a ride directly with transportation vendor.
+
+2.	The transportation dispatcher works with a variety of transportation providers to schedule a ride and dispatch the vehicle.
+
+3.	The transportation provider picks up residents at their home and drives them to and from the testing site.
+
+4.	Residents pay $2 in cash for the ride by placing it in a sealed plastic bag and leaving it in the back seat of the car. However, if a resident does not have $2, they will still get a ride, and the City will reimburse the taxi provider. 
+
+5.	Vehicles are cleaned in between each use at one of two cleaning sites the City set up with an outside cleaning vendor. 
+
+It is imperative that safety for both the driver and the passenger is prioritized in this process.  This has been accomplished through the following method:
+
+- The City provides PPE for both the driver and passenger. The driver is provided with an N-95 masks, goggles, and gloves.   The passenger is provided with a supply surgical masks and gloves.  Passengers are informed they will be required to wear a mask and gloves when they book the ride and that they will be refused a ride if they do not wear the PPE.
```

Comment:
The City provides PPE for both the driver and passenger. The driver is provided with an N-95 mask, goggles, and gloves. The passenger is provided with a supply surgical mask and gloves.  Passengers are informed that they will be required to wear a mask and gloves when they book the ride and that they will be refused a ride if they do not wear the PPE.